src/app-layout/visual-refresh/main.scss

```diff
@@ -121,7 +129,7 @@
 }
 
 // Prevent content that is visually hidden behind drawers and content area in mobile view from receiving focus
-/* stylelint-disable-next-line selector-combinator-disallowed-list, selector-max-universal */
```

Comment:
Apparently the order matters now. So it needs to ignore the universal selector first and then the combinator